packages/node_modules/@webex/plugin-meetings/test/utils/testUtils.js

```diff
@@ -191,13 +191,34 @@ const delayedTest = (callback, timeout) => new Promise((resolve) => {
   }, timeout);
 });
 
+// returns a promise that has resolve() and reject() methods
+// allowing you to resolve/reject it at any time you want
+const createControlledPromise = () => {
```

Comment:
This usually called `Defer` and it is already defined in the common package `packages/node_modules/@webex/common/src/defer.js`

Reply:
ah, nice, I wasn't aware that we had that in the SDK. Thanks for pointing it out. I've now removed `createControlledPromise()` and changed the code to use `Defer`